Formula/google-drive-upload.rb

```diff
@@ -0,0 +1,18 @@
+class GoogleDriveUpload < Formula
+  desc "Bash scripts to upload files to google drive"
+  homepage "https://github.com/labbots/google-drive-upload"
+  url "https://github.com/labbots/google-drive-upload/archive/refs/tags/v4.5.tar.gz"
+  sha256 "143bf7521fab80f5b8f5edf4180648043f900c8aa357f0b021f9ce8b06207df5"
+  license "MIT"
+  head "https://github.com/labbots/google-drive-upload.git", branch: "master"
+
+  def install
+    bin.install "release/bash/gupload"
+    bin.install "release/bash/gsync"
+  end
+
+  test do
+    assert_match "No valid arguments provided, use -h/--help flag to see usage.", shell_output("#{bin}/gupload && false", 1)
+    assert_match "No valid arguments provided, use -h/--help flag to see usage.", shell_output("#{bin}/gsync && false", 1)
```

Comment:
It doesn't have to fail so `&& false` isn't needed, but passing valid arguments would be useful

Reply:
If ` && false` don't exist, `brew test` will fail. So I have to add it. Any better method?

Reply:
Why will it fail? Because false explicitly makes the command fail, so I doubt it fails without it.

Reply:
No, `gupload` will success, and return the message `No valid ...`. However, `brew test` will fail. I found <https://github.com/Homebrew/homebrew-core/blob/master/Formula/proxychains-ng.rb> `proxychains test` will fail and `brew test proxychains-ng` will success. So I think this command must fail to make `brew test` success.

Reply:
The command in `proxychains-ng` is failing, that's why it's redirecting stderr and expecting 1 as returncode.

Reply:
Oh, I see.